assets/shaders/array_texture.wgsl

```diff
@@ -43,7 +43,7 @@ fn fragment(
         double_sided,
         is_front,
 #ifdef VERTEX_TANGENTS
-#ifdef STANDARDMATERIAL_NORMAL_MAP
+#ifdef STANDARD_MATERIAL_NORMAL_MAP
```

Comment:
This is a small drive-by fix for consistency with the instances of `STANDARD_MATERIAL` in the codebase.